scripts/auto_version.py

```
@@ -256,21 +286,64 @@ def create_release_notes(self, version: semantic_version.Version, entries: Dict[
+            
+            if category_items:
+                release_notes += f"## {category}:\n"
+                for item in category_items:
+                    # Remove the leading "- " and format consistently
                     clean_item = item[2:] if item.startswith("- ") else item
-                    release_notes += f"- {clean_item}\n"
+                    release_notes += f" - {clean_item}\n"
                 release_notes += "\n"
         
         release_notes += "---\n\n"
         release_notes += f"**Full Changelog**: [CHANGELOG.md](https://github.com/3C-SCSU/Avatar/blob/main/CHANGELOG.md)\n"
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)